schedule/yast/btrfs/btrfs_sle_libstorage-ng@hmc+ipmi.yaml

```diff
@@ -19,22 +19,16 @@ schedule:
   - installation/user_settings_root
   - installation/resolve_dependency_issues
   - installation/installation_overview
-  - installation/disable_grub_timeout
   - installation/start_install
   - installation/await_install
   - installation/logs_from_installation_system
   - installation/reboot_after_installation
-  # Called on BACKEND: ipmi
-  - '{{reconnect_mgmt_console}}'
+  - boot/reconnect_mgmt_console
+  - installation/grub_test
```

Comment:
Hmm, somehow I've completely missed that there is no `grub_test` module scheduled, so maybe it will work with disabled timeout then?

Reply:
Yes. It works.